sdk/python3/README.md

```diff
@@ -1,10 +1,15 @@
-# Confidential Cloud-Native Primitives client library for Python
+# Confidential Cloud-Native Primitives SDK for Python
 
-The Confidential Cloud-Native Primitives (CCNP) project is the solution targeted on simplify the use of Trusted Execution Environment (TEE) in cloud native environment. The service supports attestation, measurement fetching and event log collecting of various platforms including Intel Trusted Domain Extensions (TDX), Trusted Platform Modules (TPM) and AMD SEV-SNP. More platforms will be supported later.
+The Confidential Cloud-Native Primitives (CCNP) project is the solution targeted on simplifying the use of Trusted Execution Environment (TEE) in cloud-native environment. Currently, there are 2 parts included in CCNP, the services and the SDK.
+
+- Service (DaemonSet) is designed to hide the complexity of different TEE platforms and provides common interfaces and scalability for cloud-native environment.
```

Comment:
Service (DaemonSet) is -> Service is
Since there might be other deployment mode later.